samples/signer/cloudbuild-bad.yaml

```diff
@@ -0,0 +1,38 @@
+# Cloudbuild pipeline for a build with an image
+# that FAILS the vuln policy
+steps:
+  # Build a 'bad' image
+  - name: gcr.io/cloud-builders/docker
+    args:
+    - build
+    - '-t'
+    - 'gcr.io/projectgut-215417/binauthz-test'
```

Comment:
I'm assuming the projectgut usage is temporary?

Reply:
Yup